src/include/ipc/info.h

```diff
@@ -139,6 +140,15 @@ struct sof_ipc_cc_version {
 	char desc[]; /* null terminated compiler description */
 } __attribute__((packed));
 
+/* extended data: Probe setup */
+struct sof_ipc_probe_support {
+	struct sof_ipc_ext_data_hdr ext_hdr;
+
+	uint32_t probe_points_max;
+	uint32_t injection_dmas_max;
```

Comment:
Should we have some reserved fields here to cover any future updates ?

Reply:
@lgirdwood I don't see any point in it, as in I don't see Probes setup varying in a way that would require incompatible changes, but I could throw a handful of reserved fields for a good measure if You insist.

Reply:
yes please, remember "640k ought to be enough for anyone". ;)

Reply:
@lgirdwood Also I had a thought, that we probably don't need reserved fields in metadata structs, since we always report size of each given section, so kernel can always properly interpret and walk through these messages.
But I won't obstruct and add these in today.

Reply:
done